backend/src/constants.ts

```diff
@@ -17,6 +17,9 @@ export const AIRBRAKE_PROJECT_KEY = process.env.AIRBRAKE_PROJECT_KEY || 'e137655
 export const API_URL = process.env.API_URL || 'https://api.remot3.it/apv/v27'
 export const DEVELOPER_KEY = process.env.DEVELOPER_KEY || 'Mjc5REIzQUQtMTQyRC00NTcxLTlGRDktMTVGNzVGNDYxQkE3'
 
+// CLI
+export const CLI_VERSION = '0.37.6'
```

Comment:
Shouldn't this be `1.0.x`?

Reply:
Yes soon. One the works isn't released yet.